tutorials/tutorial-howto-monitor-django-prometheus.md

```diff
@@ -0,0 +1,230 @@
+# Tutorial: How to monitor a Django application with Prometheus
+
+>:TOPLIST:
+> - [Introduction](#introduction)
+> - [Prerequisites](#prereqs)
+> - [Step 0 - Set up a basic Django application (optional)](#step0)
+> - [Step 1 - Export prometheus-style monitoring metrics from your
+Django application](#step1)
+> - [Step 2 - Point Prometheus to your Django application metrics endpoint](#step2)
+> - [Step 3 - Instrument additional aspects of your application (optional)](#step3)
+> - [Next Steps](#next-steps)
+
+## Introduction [](introduction)
```

Comment:
We usually do an H1 for the title, and H3 for each section header.

Reply:
Cool, will update. Is there a style guide somewhere?